core/root/cluster.go

```diff
@@ -348,3 +366,43 @@ func (c clusterImpl) ValidateStack() (string, error) {
 
 	return strings.Join(reports, "\n"), nil
 }
+
+func printJournaldLogs(c clusterImpl, quit chan bool) error {
+	fmt.Printf("Printing filtered Journald logs for log group '%s'...\nNOTE: Due high initial entropy, failures may occur during the early stages of booting.\n", c.controlPlane.ClusterName)
```

Comment:
Excuse me but what do you mean by "high initial entropy" in this case? I'm not very confident with the language but maybe you meant "some services get failed and then restarted multiple times in the very beginning of the boot process. It is by-design"?